Packs/MicrosoftGraphListener/Integrations/MicrosoftGraphListener/MicrosoftGraphListener.yml

```diff
@@ -15,6 +15,14 @@ configuration:
   name: enc_key
```

Comment:
for the id, token and key, update the display names to be the same as in Graph Mail

Reply:
Done